Product_Display_Cards/Product_Card_04/README.md

```diff
@@ -0,0 +1,11 @@
+## Image Hover Display Card
+
+<h3> Tech Stack Used <img src = "https://media2.giphy.com/media/QssGEmpkyEOhBCb7e1/giphy.gif?cid=ecf05e47a0n3gi1bfqntqmob8g9aid1oyj2wr3ds3mg700bl&rid=giphy.gif" width = 32px> </h3>
+<p align ="left"> 
+<a href="https://www.w3.org/html/" target="_blank"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/html5/html5-original-wordmark.svg" alt="html5" width="40" height="40"/> </a> 
+<a href="https://www.w3schools.com/css/" target="_blank"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/css3/css3-original-wordmark.svg" alt="css3" width="40" height="40"/> </a>
+</p>
+
+### Preview
+
+![](https://github.com/Dezenix/frontend-html-css-js/blob/main/Product_Display_Cards/Product_Card_04/assets/Product_Display_Card.gif)
```

Comment:
embed the preview gif directly by drag and drop, also try to optimize it under 10MB

Reply:
.mov is not supported so I converted to .gif online and compressed it under 10MB
![Product-Card-Preview](https://user-images.githubusercontent.com/71960989/148538673-39d6c9e4-decb-4de2-905c-f11d71948154.gif)